src/plugin.rs

```diff
@@ -81,6 +189,7 @@ impl FmodPlugin {
         Ok(())
     }
 
+    ///todo? FMOD may also be configured to use a right-handed coordinate system by passing FMOD_INIT_3D_RIGHTHANDED to System::init.
```

Comment:
This TODO should be an issue.

Also, by using triple slash `///` you expose this to-do comment to the user. I don't think that makes sense in this case.